generators/csharp/codegen/src/ast/core/AstNode.ts

```diff
@@ -27,6 +30,24 @@ export abstract class AstNode {
             customConfig
         });
         this.write(writer);
-        return writer.toString();
+        const stringNode = writer.toString(skipImports);
+        return format ? this.toFormattedSnippet(stringNode) : stringNode;
+    }
+
+    /**
+     * function for formatting snippets, useful in testing
+     */
+    public toFormattedSnippet(code: string): string {
+        code += ";";
+        try {
+            const finalCode = AstNode.formatCSharpCode(code);
+            return finalCode;
+        } catch (e: unknown) {
+            return code;
+        }
+    }
```

Comment:
Is this still being used?

Reply:
used [here] right? (https://github.com/fern-api/fern/pull/4385/files/f61c6bd5d5f2294b376c55ee602cd3202ffefdd7#diff-37bfd52f6d61c28fe29c548d34821cbd6a98482408f2357c94f901688d260e84R34)